packages/ec-core/Cargo.toml

```diff
@@ -4,8 +4,12 @@ version = { workspace = true }
 authors = { workspace = true }
 description = { workspace = true }
 documentation = { workspace = true }
+repository = { workspace = true }
 edition = { workspace = true }
 license = { workspace = true }
+categories = ["science", "algorithms"]
+keywords = ["evolutionary-computation", "genetic", "genetic-algorithm", "framework"]
```

Comment:
I wonder if either or both of "artificial intelligence" and "machine learning" should appear here somewhere (and below).

Reply:
sure. Be aware that both these keys are limited to a max of 5 tho and that the categories are meant to match the ones on crates.io